capellambse/model/diagram.py

```diff
@@ -372,6 +368,12 @@ def __eq__(self, other: object) -> bool:
             return NotImplemented
         return self._model is other._model and self._element == other._element
 
+    @property
+    def description(self) -> str:
+        """Return the diagram description."""
+        desc = self._model._loader[self.uuid].get("documentation")
+        return desc and helpers.repair_html(desc)
```

Comment:
I'm curious on why this change is needed. The way via xmltools.HTMLAttributeProperty is not sufficient?

Reply:
No, these properties can't do something as complex as `obj._model._loader[obj.uuid]`. So we need to keep doing it this way and wait until the big refactor. :(